.pre-commit-config.yaml

```diff
@@ -83,6 +83,9 @@ repos:
           - docstring_parser>=0.16 # Match aviary pyproject.toml
           - fastapi
           - httpx < 0.28 # Match aviary pyproject.toml
+          - paper-qa>=5.12.0 # Match litqa pyproject.toml
+          - ldp>=0.20 # Match litqa pyproject.toml
+          - fh-llm-client>=0.0.11 # Match litqa pyproject.toml
```

Comment:
These only make sense for the LitQA PR, can you just open this into the LitQA branch